tsconfig.packages.json

```diff
@@ -1,5 +1,9 @@
 {
   "compilerOptions": {
+    // NOTE: For some reason, this flag has to be disabled for tsd. See:
```

Comment:
This is related to `tsd` again.

Reply:
I'm not following, the issue mentions that you must remove ` "composite": true` but I still see it in this file

Reply:
Yes, sorry this was not clear 🙈

The comment (in the `tsconfig.packages.json`) is about `tsd` and how I found out this issue with `composite`. But the `"composite": true` is needed here to build the whole monorepo.

We do disabled it on each "sub-"`package.json` for `tsd` only.